### PR TITLE
Adding missing parameters

### PR DESCRIPTION
**Pull request by get-itips**
Adding missing parameters to this article, from a list that was reported on this issue https://github.com/MicrosoftDocs/office-docs-powershell/issues/4986

From the list, the description of these can be improved:

_ConditionalAccessPolicyErrorHelpLink
BlockDownloadLinksFileType
EmailAttestationRequired
SyncPrivacyProfileProperties_

@trent-green can you help me getting a better description for those?

_AllowDownloadingNonWebViewableFiles_ has been deprecated and received an error trying to use _EnablePromotedFileHandlers_ so it is probably not ready for production use or for internal use

### CODE REVIEW

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 1]
```diff
@@ -1349,6 +1482,38 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -ConditionalAccessPolicyErrorHelpLink
+
+A Link for help when Conditional Access Policy blocks a user.
```
Comment (dariomws) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md]: We can add a comment that it should be a valid URL and a example. I have tried and if you put any string you will get "WarningConditionalAccessPolicyHelpLinkInvalidUrl".

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 2]
```diff
@@ -261,6 +291,59 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -EnableAIPIntegration
+
+This parameter enables SharePoint to process the content of files stored in SharePoint and OneDrive with sensitivity labels that include encryption. (Currently in public preview). For more information, see [Enable sensitivity labels for Office files in SharePoint and OneDrive (public preview)](https://docs.microsoft.com/microsoft-365/compliance/sensitivity-labels-sharepoint-onedrive-files?view=o365-worldwide)
```
Comment (dariomws) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md]:
```suggestion
This parameter enables SharePoint to process the content of files stored in SharePoint and OneDrive with sensitivity labels that include encryption. (Currently in public preview). For more information, see [Enable sensitivity labels for Office files in SharePoint and OneDrive (public preview)](https://docs.microsoft.com/microsoft-365/compliance/sensitivity-labels-sharepoint-onedrive-files).
```

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 3]
```diff
@@ -261,6 +291,59 @@ Accept pipeline input: False
...
+```
+
+### -MarkNewFilesSensitiveByDefault
+
+If external sharing is turned on, sensitive content could be shared and accessed by guests before the Office DLP rule finishes processing, you can address this issue by configuring this parameter.
+Possible values are
+- BlockExternalSharing: Prevents guests from accessing newly added files until at least one Office DLP policy scans the content of the file.
+- AllowExternalSharing: Disables this feature.
+
+For more information see [Mark new files as sensitive by default](https://docs.microsoft.com/sharepoint/sensitive-by-default)
```
Comment (dariomws) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md]:
```suggestion
For more information see [Mark new files as sensitive by default](https://docs.microsoft.com/sharepoint/sensitive-by-default).
```

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 4]
```diff
@@ -770,6 +871,22 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -dd
```
Comment (dariomws) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md]: I couldn't find this parameter. Does this exist?

**Comment by dariomws** [comment]
I think you forget to add a description for FilePickerExternalImageSearchEnabled.

Maybe we can add "This parameter is reserved for internal Microsoft use" for AllowDownloadingNonWebViewableFiles and EnablePromotedFileHandlers to avoid questions about them until they are enabled.

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md (hunk 4 above)
Comment (get-itips) [reply sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md]: It is a typo from editing, will fix now. thanks

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md (hunk 1 above)
Comment (get-itips) [reply sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md]: done, thanks

**Comment by get-itips** [comment]
> Maybe we can add "This parameter is reserved for internal Microsoft use" for AllowDownloadingNonWebViewableFiles and EnablePromotedFileHandlers to avoid questions about them until they are fully available.

As AllowDownloadingNonWebViewableFiles is deprecated it will probably removed soon, so, is it worth adding it to the article?

Added EnablePromotedFileHandlers. Thanks

**Comment by dariomws** [comment]
> > Maybe we can add "This parameter is reserved for internal Microsoft use" for AllowDownloadingNonWebViewableFiles and EnablePromotedFileHandlers to avoid questions about them until they are fully available.
> 
> As AllowDownloadingNonWebViewableFiles is deprecated it will probably removed soon, so, is it worth adding it to the article?
> 
> Added EnablePromotedFileHandlers. Thanks

Good question... I think it's there so we can add the information to let the reader know that they shouldn't use it... But if it will be removed soon I think it doesn't have sense.

So, it's your desition...

**Comment by tseward** [comment]
In general, if there is a parameter available, it should be presented in the cmdlet documentation.

**Pull request merged**